mr/models/model_artifact_create.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+from dataclasses import dataclass, field
+from kiota_abstractions.serialization import Parsable, ParseNode, SerializationWriter
+from typing import Any, Callable, Dict, List, Optional, TYPE_CHECKING, Union
+
+if TYPE_CHECKING:
+    from .with_base_artifact_update import WithBaseArtifactUpdate
+
+from .with_base_artifact_update import WithBaseArtifactUpdate
+
+@dataclass
+class ModelArtifactCreate(WithBaseArtifactUpdate):
```

Comment:
This model seems to be missing properties from `WithBaseResourceCreate`, `WithBaseResourceUpdate` and `WithModelArtifactUpdate`.

`with_base_resource_create` and `with_model_artifact_update` seem to be missing completely too.